package.json

```diff
@@ -39,21 +39,26 @@
     "discord-oauth2": "^2.11.0",
     "dotenv": "^16.0.3",
     "dotenv-flow": "^3.2.0",
+    "envalid": "^7.3.1",
```

Comment:
This is used to validate environment variables.  It's nice to use so that if we're missing an environment variable, we'll know immediately (because it will error) instead of hitting a weird error because some value is `undefined`.  More info [here](https://github.com/af/envalid#readme)